### DISCUSSION TITLE
Please make the button to open the menu another button.

### DISCUSSION THREAD

**Discussion by FallingHoglin** [Draylar/identity#457] [Ideas]
I can not open the menu because the button just does not want, why is it the: ` button? It is wierd.
My suggestion is that it becomes a part of the keybinds control, so the players can change it into any button they want.

**Comment by Natedogwoooof** [comment]
Change it in the controoles menu to whatever.

**Comment by FallingHoglin** [comment]
I have tried, but I could never find it.

ons. 25. jan. 2023 16.46 skrev Natedogwoooof ***@***.***>:
